### ISSUE TITLE
Error: ImportError: No module named pyvirtualdisplay

### ISSUE THREAD

**Issue by jrvansuita** [ahmadudin/electron-instaPy-GUI#13]
Im facing Error: ImportError: No module named pyvirtualdisplay(3.png) on mac. I have python 2.7, 3.5 and 3.6 installed. (1.png)

Already installed pyvirtualdisplay for all versions.
pip3.6 install pyvirtualdisplay

--------------------

Requirement already satisfied: pyvirtualdisplay in /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages
Requirement already satisfied: EasyProcess in /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages (from pyvirtualdisplay)

Sometimes, when I open Instapy Gui, no error message is shown. But when I click run, nothing happens, likes image 2.png.
<img width="583" alt="1" src="https://user-images.githubusercontent.com/12175215/31857220-83193afe-b6b7-11e7-9c67-04306f884add.png">
![Uploading 2.png…]()
![Uploading 3.png…]()

**Comment by ahmadudin** [comment]
the gui just simply displaying error thrown by python, have you try to run instapy on terminal directly?

**Comment by jrvansuita** [comment]
@ahmadudin Yeap. On terminal it's working normally. That's weird.

**Comment by ahmadudin** [comment]
The command used in GUI is simply `python quickstart.py`, is it the python version that you installed pyvirtualdriver on?

**Comment by jrvansuita** [comment]
Yes, If I type `python quickstart.py` on terminal it works. If I click run on InstaPy Gui, shows me a message Error: ImportError: No module named pyvirtualdisplay

**Comment by JacobSpruce** [comment]
Can I edit command in GUI to `python3 quickstart.py`? Because I get error in terminal with default comand `python quickstart.py`. I guess versions interaction makes this error somehow. [OS X]

**Comment by bunyadahmadli** [comment]
Bu hata düzeltildimi?
